drivers/hmis/app/models/hmis/user.rb

```diff
@@ -65,10 +65,15 @@ def load_effective_permissions
       raise "Invalid entity '#{entity.class.name}'" unless joins_association.present?
       return false unless send("#{permission}?")
 
+      access_group_ids = Hmis::GroupViewableEntity.includes_project(entity).pluck(:access_group_id) if entity.is_a?(Hmis::Hud::Project)
+      access_group_ids = Hmis::GroupViewableEntity.includes_organization(entity).pluck(:access_group_id) if entity.is_a?(Hmis::Hud::Organization)
+      access_group_ids = Hmis::GroupViewableEntity.includes_data_source(entity).pluck(:access_group_id) if entity.is_a?(GrdaWarehouse::DataSource)
+      access_group_ids = Hmis::GroupViewableEntity.includes_project_access_group(entity).pluck(:access_group_id) if entity.is_a?(GrdaWarehouse::ProjectAccessGroup)
```

Comment:
These 2 can be removed. We will only ever need to check access at the project and org level. 90% of the time will be using project. Organization is needed for the "can edit organization" type permissions. There is no Data Source or Project Group editing in the HMIS, so we shouldn't need those.